olp-cpp-sdk-authentication/src/SignInResultImpl.cpp

```diff
@@ -91,5 +103,63 @@ const std::string& SignInResultImpl::GetScope() const { return scope_; }
 
 bool SignInResultImpl::IsValid() const { return is_valid_; }
 
+std::string SignInResultImpl::ParseJwtToken(const std::string& token) {
+  std::string result;
+  std::string::size_type first_dot = token.find_first_of('.');
```

Comment:
```suggestion
const std::string::size_type first_dot = token.find_first_of('.');
```
Prefer to declare the variables const by default.